README.md

```diff
@@ -1,2 +1,161 @@
-# tami
-IDE and engine for text-quest games
+# Tami
+
+Engine + IDE for text-quest games. Something similar to RenPy or rather Ink.
+
+### IDE
+
+At the left side there is a list of all files, locations/actions, variables, inventory items and persons for dialogs.  
+At the center is one big text area with syntax highligtinting for current selected file of script.  
+At the right side there are setup script (in this field allowed only `+{item}`, `variable = number` or `// command`), callstack widget, log and game realtime preview.
+
+All script files must be in `script` folder and have `.tami` extension. And `setup.tami` for setup script. If `debug_mode` is on, near each `script.tami` will be generated `script-debug.txt` just for debug.
+
+---
+
+### Syntax:
+
+Sentences:
+
+Any string that not a part of other syntax constuctions is a sentence  
+It just appears on the screen one by one instantly  
+Can contains links to actions - `[some action]` or `[some action (display name)]`  
+Can contains also interpolated variables - `[:var_name:]`  
+All text is passing to screen as HTML, so you can use any html tags and styles
+
+Dialogs:
+
+First, you need to define character with display name - `character mom = My mom`  
+Then you can use `mom` as person in dialogs  
+`person: Sentense` or `person: (additional info) Sentense`  
+Each message will be displayed separately and blocks side-panel until player clicks  
+Can contains also interpolated variables - `[:var_name:]`
+
+Menus:
+
+```
++ First choice:
+	What to do if player click at First choice
++ Second choice:
+	What to do if player click at First choice
+```
+
+Menu blocks all space while on screen
+
+Actions:
+
+```
+> action name:
+	Some block of text or commands
+	.
+```
+
+In any block you can get current active action from variable `action`:  
+`0` - look  
+`1` - interact  
+`2` - apply something from the inventory  
+Also variables `LOOK`, `INTERACT` and `USE` set to `0` or `1` accordingly to variable `action`  
+Any action must ends up with `.` - analog of `return` in programming languages, because action called by `call`, not `jump` (detailed information below)
```

Comment:
```suggestion
Any action must end up with `.` - analog of `return` in programming languages, because actions are called by `call`, not `jump` (see detailed information below)
```

Also would be nice to attach an anchor link (like `#call-vs-jump`).